infoflow/tests/test_entropy_kde.py

```diff
@@ -0,0 +1,37 @@
+from infoflow import entropy_kde as kde
+from infoflow import multidimkde as multkde
+import numpy as np
+import os
+
+path = os.path.dirname(os.path.realpath(__file__))
+
+
+# tests first case of matlab demo script.
+def test_basic():
+    x = np.genfromtxt(path + '/input_kde/entropykdeinputx.txt', delimiter="\t")
+    y = np.genfromtxt(path + '/input_kde/entropykdeinputy.txt', delimiter='\t')
+    n = 10
+    timelagx = 2
+    timelagy = 2
+    bw_coeff = 1
+    ans = kde.kerneldensityestimation(x, y, timelagx, timelagy, n, bw_coeff)
+    result = 0.798553577280728
```

Comment:
These kinds of tests are very bad at hitting corner cases.  What you want is to figure out how your algorithm responds to all extreme inputs and parameters, and work out minimal test cases for those examples.  Working out examples by hand are advisable!